manim/mobject/three_d/three_dimensions.py

```diff
@@ -926,6 +926,10 @@ def __init__(
     ):
         self.thickness = thickness
         self.resolution = (2, resolution) if isinstance(resolution, int) else resolution
+
+        start = np.array(start, np.float64)
+        end = np.array(end, np.float64)
```

Comment:
```suggestion
        start = np.array(start, dtype=np.float64)
        end = np.array(end, dtype=np.float64)
```
nitpick: I prefer using keyword arguments for dtype